parts/k8s/cloud-init/artifacts/cse_config.sh

```diff
@@ -429,7 +432,11 @@ ensureJournal() {
   systemctlEnableAndStart systemd-journald || exit {{GetCSEErrorCode "ERR_SYSTEMCTL_START_FAIL"}}
 }
 installKubeletAndKubectl() {
-  if [[ ! -f "/usr/local/bin/kubectl-${KUBERNETES_VERSION}" ]]; then
+  path=/usr/local/bin
+  if [[ $OS == $FLATCAR_OS_NAME ]]; then
+    path=/opt
```

Comment:
@invidian is there a reason we're installing kubelet and kubectl in /opt/ instead of /opt/bin? Should we change this to /opt/bin?